Gordon360/Services/ServiceInterfaces.cs

```diff
@@ -273,10 +273,12 @@ public interface IHousingService
         bool CheckIfHousingAdmin(string userId);
         bool AddHousingAdmin(string id);
         bool RemoveHousingAdmin(string id);
-        int SaveApplication(string editorId, string sess_cde, string [] applicants, ApartmentChoiceViewModel[] apartmentChoices);
-        int EditApplication(string editorId, string sess_cde, int apartAppId, string [] newApplicantIDs, ApartmentChoiceViewModel[] newApartmentChoices);
-        bool ChangeApplicationEditor(string editorId, int apartAppId, string newEditorId);
+        int SaveApplication(string userID, string sess_cde, string editorID, ApartmentApplicantViewModel[] apartmentApplicants, ApartmentChoiceViewModel[] apartmentChoices);
+        int EditApplication(string userID, string sess_cde, int applicationID, string newEditorID, ApartmentApplicantViewModel[] newApartmentApplicants, ApartmentChoiceViewModel[] newApartmentChoices);
+        bool ChangeApplicationEditor(string userID, int applicationID, string newEditorId);
```

Comment:
That was not a mistake, What is the reason for using apartAppID in the local variables/parameters as opposed to applicationID?

Reply:
It was uncoordinated naming, they should all be either one or the other. Here it seemed you were doing both, but I guess you were just following suit.

Reply:
The database names tend to be limited by what makes sense for the SQL syntax, whereas the local variables are mostly designed to play nicely with the frontend, so I made sense to make the variables and the view models match a scheme that makes sense from an "outside talking to the API" perspective, and it seemed kind of unnecessary to specify that the application ID is an "apartment" application ID, since no other application IDs will ever exist within the same scope. 
There was also a lot of inconsistency between the use of Aprt vs. Apart, which both felt somewhat non-obvious and/or redundant,

The since all the SQL query return types are now mapped to the correct models from the EDMX rather than using hand-made view models, the only remaining view models are the ones used to represent the data that goes to/from the frontend, so it seemed logical to me to make the naming scheme a little less ambiguous and styled to match many of the other parts of the project

Reply:
Yeah, I agree. Do you have a plan in mind for that naming scheme, or is this an invitation to help you decide?

Reply:
The changes I have made so far have already begun that conversion. I believe I also began the switch of variable names from "ID" to "username", which still works even if we haven't updated the database yet to match since both things are stored as strings.